osbs/build/user_params.py

```diff
@@ -162,36 +162,79 @@ def attrs_finalizer(self):
                         'Two user params with the same name: {}'.format(param.name))
                 self.convert_dict[param.name] = param
 
-    def set_params(
-        self,
-        build_from=None,
-        build_image=None,
-        build_imagestream=None,
-        component=None,
-        koji_target=None,
-        koji_task_id=None,
-        orchestrator_deadline=None,
-        platform=None,
-        reactor_config_map=None,
-        reactor_config_override=None,
-        scratch=None,
-        signing_intent=None,
-        user=None,
-        worker_deadline=None,
-        **kwargs
-    ):
+    def set_params(self,
+                   build_conf=None,
+                   build_from=None,
```

Comment:
why are we allowing still : build_from, build_image, build_imagestream params??
they are read from build_conf now

Reply:
also :
orchestrator_deadline
worker_deadline
reactor_config_map

those are read from build_conf so no need to allow pass it as params as well

Reply:
the test cases work with build_conf=None, rewriting them is a huge pain which I will do after this merges.